gulp/tasks/clean.js

```diff
@@ -1,12 +1,21 @@
 const configClean = require( '../config' ).clean;
```

Comment:
Let's get rid of the references to config altogether and remove the `clean` entry from `config.js`. All it is is a obfuscating shell over paths. `paths.processed + '/css/**/*'` is clearer than `configClean.css + '/**/*'` in my opinion. The only place for config settings is when they are used in multiple places.

Reply:
Sounds great to me. Frequently tabbing over to the config file to look up paths is tedious.